apps/web/src/components/team-builder/v2/builder.module.css

```
@@ -934,27 +949,127 @@
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-shrink: 0;
+  width: 220px;
+  background: color-mix(in oklch, var(--muted) 40%, transparent);
+  border-left: 1px solid var(--border);
+  padding: 8px 0;
+  animation: calcColEnter 0.3s cubic-bezier(0.4, 0, 0.2, 1) both;
+}
+
+@keyframes calcColEnter {
```

Stylelint (17.9.0)

[error] Expected keyframe name "calcColEnter" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)